### DISCUSSION TITLE
Mongo库使用？

### DISCUSSION THREAD

**Discussion by terry8210** [cloudwu/skynet#1746] [Q&A]
Mongo库的使用问题，如何只更新文档中的某个字段的子字段数据？
原始数据：
{name="123",base={sex=1,level=1}}
目标数据
{name="123",base={sex=2,level=1}}

**Comment by hanxi** [comment]
试试这样 {['base.sex']=2}

**Reply by terry8210** [reply]
可以，这种方式行，谢谢！
